build_system/clerk.ml

```diff
@@ -461,7 +461,9 @@ let collect_all_ninja_build (ninja : ninja) (tested_file : string) (reset_test_o
       List.fold_left
         (fun (ninja, test_names) expected_output ->
           match expected_output.backend with
-          | Cli.Interpret | Cli.Proof | Cli.Typecheck | Cli.Dcalc | Cli.Scopelang ->
+          (* (Emile) NOTE: Scopelang <> Scalc ?*)
```

Comment:
Yes "Scopelang" stands for the scope language and "Scalc" for the statement calculus, see the architecture diagram here: https://catala-lang.org/ocaml_docs/catala/index.html